openslides/topics/models.py

```diff
@@ -17,19 +18,26 @@ class Topic(RESTModelMixin, models.Model):
     text = models.TextField(blank=True)
     attachments = models.ManyToManyField(Mediafile, blank=True)
 
+    # In theory there could be one then more agenda_item. But support only one.
+    # See the property agenda_item.
+    agenda_items = GenericRelation(Item)
+
     class Meta:
         default_permissions = ()
 
     def __str__(self):
         return self.title
 
+    @classmethod
+    def all_query(cls):
+        return cls.objects.prefetch_related('attachments', 'agenda_items')
+
```

Comment:
@normanjaeckel Please see my new commit. I was not able to write add a manager via the RestModel. Maybe it is a django bug, but a Manager added via a Mixin does not work at all. If you succeed, then please let me know.

Do you think it is ok to rewrite the default queryset? I don't see any problem.